pkg/metrics/smart/metrics.go

```diff
@@ -169,18 +326,42 @@ func (m *Metrics) SetBlockDeviceCurrentTemperature(currentTemp int16) *Metrics {
 	m.blockDeviceCurrentTemperature.WithLabelValues(m.UUID,
 		m.Path,
 		m.HostName,
-		m.NodeName).
+		m.NodeName,
+	).
 		Set(float64(currentTemp))
 	return m
 }
 
+// SetBlockDeviceHighestTemperature sets the highest temperature value to the metric
+func (m *Metrics) SetBlockDeviceHighestTemperature(highTemp int16) *Metrics {
+	m.blockDeviceHighestTemperature.WithLabelValues(m.UUID,
+		m.Path,
+		m.HostName,
+		m.NodeName,
+	).
+		Set(float64(highTemp))
```

Comment:
Can you check for `gofmt` errors here. I think the `Set(` method is misaligned.

Reply:
The same comment applies for all the `Set()` methods.

Reply:
Didn't get it @akhilerm . Which way are they misaligned? Already ran go fmt on this.

Reply:
nvm, I thought `Set(` and `).` from line 341, should be at same level of indentation.